### PR TITLE
clarify licensing

### PR DESCRIPTION
**Pull request by gnunicorn**
refs #4753 .

This removes an old deprecated dependency, fixing an unknown license leaking in; and sets the `license`-field in our local `Cargo.toml` files (to `GPL-3.0` as our code already states). Except for `node-template`, which is set to be the `Unlicense`-License.

### CODE REVIEW

**Comment by gnunicorn** [comment]
The check-runtime test failing is a false-flag, no code has been changed... but to make the CI pass, I can bump that version, too...

File: bin/node-template/runtime/Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name = "node-template-runtime"
 version = "2.0.0"
 authors = ["Anonymous"]
 edition = "2018"
+license = "GPL-3.0"
```
Comment (bkchr) [inline bin/node-template/runtime/Cargo.toml]:
```suggestion
license = "Unlicense"
```

File: client/executor/runtime-test/Cargo.toml [hunk 2]
```diff
@@ -4,8 +4,10 @@ version = "2.0.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 build = "build.rs"
+license = "GPL-3.0"
 
 [dependencies]
+
```
Comment (bkchr) [inline client/executor/runtime-test/Cargo.toml]:
```suggestion
```

File: test-utils/runtime/Cargo.toml [hunk 3]
```diff
@@ -4,8 +4,10 @@ version = "2.0.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 build = "build.rs"
+license = "GPL-3.0"
 
 [dependencies]
+
```
Comment (bkchr) [inline test-utils/runtime/Cargo.toml]:
```suggestion
```

**Comment by bkchr** [comment]
> The check-runtime test failing is a false positive, no code has been changed... but to make the CI pass, I can bump that version, too...

You don't need to.

File: client/executor/runtime-test/Cargo.toml (hunk 2 above)
Comment (DemiMarie-parity) [inline client/executor/runtime-test/Cargo.toml]: nit: extra newline

File: primitives/runtime-interface/test-wasm/Cargo.toml [hunk 4]
```diff
@@ -4,8 +4,10 @@ version = "2.0.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 build = "build.rs"
+license = "GPL-3.0"
 
 [dependencies]
+
```
Comment (DemiMarie-parity) [inline primitives/runtime-interface/test-wasm/Cargo.toml]: Nit: extra newline

File: test-utils/runtime/Cargo.toml (hunk 3 above)
Comment (DemiMarie-parity) [inline test-utils/runtime/Cargo.toml]: Nit: extra newline

File: bin/utils/chain-spec-builder/Cargo.toml [hunk 5]
```diff
@@ -4,8 +4,10 @@ version = "2.0.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 build = "build.rs"
+license = "GPL-3.0"
 
 [dependencies]
+
```
[the remark this answers is not in the document]
Comment (gnunicorn) [reply bin/utils/chain-spec-builder/Cargo.toml]:
```suggestion
```

File: primitives/runtime-interface/test-wasm/Cargo.toml (hunk 4 above)
Comment (gnunicorn) [reply primitives/runtime-interface/test-wasm/Cargo.toml]:
```suggestion
```

**Pull request merged**